### PR TITLE
Update es.po

### PR DESCRIPTION
**Pull request by MagicFab**
MIssing spaces, small terms corrections, typos

### CODE REVIEW

File: po/es.po [hunk 1]
```diff
@@ -88,7 +88,7 @@ msgstr "Firmware"
 
 #: lib/logitech_receiver/i18n.py:48
 msgid "Hardware"
-msgstr "Hardware"
+msgstr "Material"
```
Comment (hobarrera) [inline po/es.po]: As a native Spanish speaker, I'd like to point out that the word "hardware" is pretty much used every day, and this is the first time I've seen "Material" as a translation of hardware. I'm pretty sure this'd be very confusing for users.

**Comment by hobarrera** [comment]
Aside from my above comment, this seems like a pretty good improvement. :)

File: po/es.po (hunk 1 above)
Comment (MagicFab) [inline po/es.po]: o_O - I am not sure what happened there, I fully agree about "Hardware"! Feel free to revert that change, and thanks for your other comment.

Comment (hobarrera) [inline po/es.po]: > Feel free to revert that change

I'm not a solaar dev, just a user/contributor who keeps a close eye. :)

**Pull request merged**